addons/im_livechat/static/src/embed/common/store_service_patch.js

```diff
@@ -13,6 +13,10 @@ patch(Store.prototype, {
         const livechatService = this.env.services["im_livechat.livechat"];
         if (livechatService.state === SESSION_STATE.PERSISTED) {
             await super.initialize();
+            livechatService.thread ??= this.store.Thread.get({
+                id: livechatService.savedState?.store["discuss.channel"][0].id,
+                model: "discuss.channel",
+            });
```

Comment:
Why not put this code in `livechatService.initialize()`? This is a concern of livechat service, not store service.

Reply:
Live chat service does not trigger the initialization of the store service. It just prevent it if not required. Here is the only place we can be sure the store was initialized so the good place to restore our thread or to discard it if it is outdated.

Reply:
Ok for now.

Still feels weird that livechat service is initialized before store: store is a core feature so should be initialized before livechat service instead. And preventing usual store initialization feels off, but we are aware of this weirdness in code and will clean later